docker_registry/images.py

```diff
@@ -60,6 +62,16 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+def valid_image_id(f):
```

Comment:
Well, this is pretty low-effort as far as improvements go, do you mind if I amend the PR with that particular change? It's not like we're getting rid of this code anytime soon.